src/main/java/picard/analysis/CollectRrbsMetrics.java

```diff
@@ -127,6 +123,20 @@
 
     private static final Log log = Log.getInstance(CollectRrbsMetrics.class);
 
+    // return a custom argument collection since this tool uses a (required) argument name
+    // of "REFERENCE", not "REFERENCE_SEQUENCE"
+    @Override
+    protected ReferenceArgumentCollection getReferenceArgumentCollection() {
+        return new CollectRrbsMetricsReferenceArgumentCollection();
+    }
+
+    public static class CollectRrbsMetricsReferenceArgumentCollection implements ReferenceArgumentCollection {
+        @Argument(doc = "The reference sequence fasta file", shortName = StandardOptionDefinitions.REFERENCE_SHORT_NAME)
+        public File REFERENCE;
```

Comment:
Barclay doesn't directly support either of these now, but I think it could be added if we wanted to go down this path since the underlying jopt library appears to support both abbreviations and multiple names.